src/containers/TermsOfUse/index.tsx

```diff
@@ -0,0 +1,330 @@
+          <li>use another user's account without permission;</li>
+          <li>provide false or inaccurate information when registering an account;</li>
+          <li>interfere or attempt to interfere with the proper functioning of the Service;</li>
+          <li>
+            make any automated use of the system, or take any action that we deem to impose or to potentially impose an
+            unreasonable or disproportionately large load on our servers or network infrastructure;
+          </li>
+          <li>
+            bypass any robot exclusion headers or other measures we take to restrict access to the Service or use any
+            software, technology, or device to scrape, spider, or crawl the Service or harvest or manipulate data
+          </li>
+          <li>
+            publish or link to malicious content intended to damage or disrupt another user's browser or computer.
+          </li>
+        </ul>
+
+        <h2 className="TermsOfUse__subtitle">Posting and Conduct Restrictions</h2>
+        <p>
+          When you create your own personalized account, you may be able to provide ("User Content"). You are solely
+          responsible for the User Content that you post, upload, link to or otherwise make available via the Service.
+          You agree that we are only acting as a passive conduit for your online distribution and publication of your
+          User Content. The Company, however, reserves the right to remove any User Content from the Service at its
+          discretion. The following rules pertain to User Content. By transmitting and submitting any User Content while
+          using the Service, you agree as follows:
+        </p>
+        <ul className="TermsOfUse__list">
+          <li>
+            You are solely responsible for your account and the activity that occurs while signed in to or while using
+            your account;
+          </li>
+          <li>You will not post information that is malicious, false or inaccurate;</li>
+          <li>
+            You will not submit content that is copyrighted or subject to third party proprietary rights, including
+            privacy, publicity, trade secret, etc., unless you are the owner of such rights or have the appropriate
+            permission from their rightful owner to specifically submit such content; and
+          </li>
+          <li>
+            You hereby affirm we have the right to determine whether any of your User Content submissions are
+            appropriate and comply with these Terms of Service, remove any and/or all of your submissions, and terminate
+            your account with or without prior notice.
+          </li>
+        </ul>
+        <p>
+          You understand and agree that any liability, loss or damage that occurs as a result of the use of any User
+          Content that you make available or access through your use of the Service is solely your responsibility. The
```

Comment:
or access through your use of the Service => , or access through your use of the Service,